src/modules/UI.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable linebreak-style */
+/* eslint-disable prefer-const */
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As stated before kindly remove theses lines and fix the linters and you can set up linters locally and run this command `npx eslint . --fix` to fix them.